.github/workflows/test.yml

```diff
@@ -12,20 +12,66 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.42
+          args: --timeout=5m
+
   test:
     runs-on: ubuntu-latest
     steps:
+      # setup indy ledger
+      - uses: actions/checkout@v2
+        with:
+          repository: "bcgov/von-network"
+          ref: "92e4960b7b36cf06381184f8df4ef59d137145eb"
+      - name: skip setting volumes for web server
+        run: |
+          sed '/      - .\/config:\/home\/indy\/config/d' docker-compose.yml > docker-compose.yml.tmp
+          sed '/      - .\/server:\/home\/indy\/server/d' docker-compose.yml.tmp > docker-compose.yml
+
+      - name: start indy ledger
+        run: |
+          ./manage build
+          ./manage start
+
       - name: setup
         uses: actions/setup-go@v2
         with:
           go-version: 1.16.x
       - name: checkout
         uses: actions/checkout@v2
       - name: install indy
-        run: make indy_to_debian
+        run: |
+          make indy_to_debian
+          sudo apt-get install -y indy-cli
```

Comment:
I was just wondering what `indy-cli` is used and seems that for the pool creation. I suppose using our `findy-agent` would be overkill for that and installation is not so easy. This is just a stupid comment 🤣

Reply:
Yes, would have used our cli for this but as it doesn't have any indy-specific features anymore, thought that this would be better than introducing "dependency" to findy-agent. Of course there could be a tiny dedicated go-program for this as well, or the pool handle creation could be part of the test set. But let's start by using indy-cli and improve later if needed.